Platform/QemuBoardPkg/Script/TestCases/firmware_update.py

```diff
@@ -257,7 +257,7 @@ def main():
     cmd = [ sys.executable,
             'BootloaderCorePkg/Tools/GenCapsuleFirmware.py',
             '-p',  'BIOS', bios_img,
-            '-k',  'BootloaderCorePkg/Tools/Keys/TestSigningPrivateKey.pem',
+            '-k',  '../SlimbootKeyDir/FirmwareUpdateTestKey_Priv_RSA2048.pem',

```

Comment:
Why is the default path for keys changed from BootloaderCorePkg/Tools/Keys to SlimbootKeyDir/ ?

Reply:
BootloaderCorePkg/Tools/Key would would have keys.
Generated keys likely to be in ../SlimbootKeyDir/. 
Instead of path we can use FIRMWAREUPDATE_KEY_ID.  Due to env setting issues in travis builds i have used direct key path.

Reply:
Maybe just call "SblKeys"  for the dir ?

Reply:
Yes. Changed SlimbootKeyDir to SblKey which is an default keys folder